tests/Predis/Command/Redis/BITOP_Test.php

```diff
@@ -181,15 +182,15 @@ public function testBitwiseNOTAcceptsOnlyOneSourceKey(): void
     }
 
     /**
-     * @group connected
-     * @requiresRedisVersion >= 2.6.0
+     * @group disconnected
```

Comment:
@htemelski-redis could we also add a test with new BITOP operators? Is it already released in 8.0.2?